tests/rptest/services/redpanda.py

```diff
@@ -785,10 +785,12 @@ def raise_on_bad_logs(self, allow_list=None):
                 f"Scanning node {node.account.hostname} log for errors...")
 
             for line in node.account.ssh_capture(
-                    f"grep -e ERROR -e Segmentation\ fault -e [Aa]ssert {RedpandaService.STDOUT_STDERR_CAPTURE}"
+                    f"grep -e ERROR -e Segmentation\ fault -e [Aa]ssert {RedpandaService.STDOUT_STDERR_CAPTURE} || true"
             ):
                 line = line.strip()
 
+                assert "No such file or directory" not in line
+
```

Comment:
This is so we still get a failure if the service output is missing, but we don't get an error if we fail to match.. Makes sense.